The make install command use an unexisting instruction

The make install command lauch a make db-create command that don't exists. 

The error message we get : 
```
make db-create
make[1]: Entering directory '/srv/bolt'
make[1]: *** No rule to make target 'db-create'.  Stop.
make[1]: Leaving directory '/srv/bolt'
make: *** [Makefile:9: install] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,7 @@ help:
 
 install: ## to install all project
 	composer install
-	make db-create
+	make db-reset
 
 server: ## to start server
 	bin/console server:start 127.0.0.1:8088 || true

